sql/pgwire/types.go

```diff
@@ -238,19 +238,19 @@ func formatTs(t time.Time) (b []byte) {
 
 var (
 	oidToDatum = map[oid.Oid]parser.Datum{
-		oid.T_bool:      parser.DummyBool,
-		oid.T_bytea:     parser.DummyBytes,
-		oid.T_date:      parser.DummyDate,
-		oid.T_float4:    parser.DummyFloat,
-		oid.T_float8:    parser.DummyFloat,
-		oid.T_int2:      parser.DummyInt,
-		oid.T_int4:      parser.DummyInt,
-		oid.T_int8:      parser.DummyInt,
-		oid.T_interval:  parser.DummyInterval,
-		oid.T_numeric:   parser.DummyDecimal,
-		oid.T_text:      parser.DummyString,
-		oid.T_timestamp: parser.DummyTimestamp,
-		oid.T_varchar:   parser.DummyString,
+		oid.T_bool:        parser.DummyBool,
+		oid.T_bytea:       parser.DummyBytes,
+		oid.T_date:        parser.DummyDate,
+		oid.T_float4:      parser.DummyFloat,
+		oid.T_float8:      parser.DummyFloat,
+		oid.T_int2:        parser.DummyInt,
+		oid.T_int4:        parser.DummyInt,
+		oid.T_int8:        parser.DummyInt,
+		oid.T_interval:    parser.DummyInterval,
+		oid.T_numeric:     parser.DummyDecimal,
+		oid.T_text:        parser.DummyString,
+		oid.T_timestamptz: parser.DummyTimestamp,
```

Comment:
Can clients still send `T_timestamp`? Wondering if you should be maintaining `T_timestamp` support here and in `decodeOidDatum`.